.editorconfig

```
@@ -0,0 +1,13 @@
+root = true
+
+[*.js]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.js]
+indent_style = space
+indent_size = 2
```

```suggestion
indent_size = 4
```

C'est une préférence personnelle ^^